fs/fs.go

```diff
@@ -2,14 +2,17 @@
 package fs
 
 import (
+	"bufio"
 	"fmt"
+	"io"
 	"os"
 	"os/user"
 	"path"
 )
 
 const defaultDirectoryPermission = 0740
 const rwFilePermission = 0600
+const copyChunkSize = 1024
```

Comment:
The main point of buffering is to not hammer the disk on read/write. This is even for the case when your source or dest actually refuse to read as much as you request. Do you know how reading a File works when the Filesystem is actually EBS and network-based? Does it return all bytes asked? Does go handle that for us? I have no idea. But that is the point.

Now CopyBuffer works I guess to hammer the disk less, assuming the file reader/writers do behave properly.

Now, if you are going to manually specify the buffer, then don't make it worse than the default. which is 32KB and not 1KB:

https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/io/io.go;l=412

I was hoping we could copy in something larger than 32KB chunks. We should definitely not be copying in 1KB chunks (!).

Sorry if it sounds pendantic, but my problem is that there seems to be 0 attention to these things. Did I not say at the beginning that I'd prefer if we write in larger chunks? What am I supposed to say when the result is we now want to write in chunks 32 times smaller?

Reply:
Hi @hsanjuan, I have updated the value to 128KB. What do you think now?